CSK_Module_MultiDeepLearning/project.mf.xml

```diff
@@ -224,6 +238,17 @@ Needs then to be called via "Script.callFunction()" (INFO: By using this, the fi
                     <desc>Get amount of created DeepLearning instances.</desc>
                     <return desc="Amount" multiplicity="1" name="amount" type="int"/>
                 </function>
+                <function name="processImageWithScoresNUM">
+                    <desc>Example of dynamically served function to process DNN on an image and return the scores for each class. +
```

Comment:
What#s DNN? keep abreviations to a minimum

Reply:
Thx for the review.
Issues should be fixed now. (DNN = Deep Neural Network)